lib/vsc/utils/fancylogger.py

```diff
@@ -431,7 +438,7 @@ def getRootLoggerName():
         return "not available in optimized mode"
 
 
-def logToScreen(enable=True, handler=None, name=None, stdout=False):
+def logToScreen(enable=True, handler=None, name=None, stdout=False, color=False):
```

Comment:
a single type (and None).
so either strings/None or booleans/None. with boolean/None you are a bit limited, so True=always color/False=never color/None=auto is possible, but there' can't be a 4th option in the future.
if you use strings, None could be set equal to 'no/never'